src/raftstore/store/peer.rs

```diff
@@ -966,6 +1047,19 @@ impl Peer {
                 // apprently, all the callbacks whose term is less than entry's term are stale.
                 self.notify_not_leader(cmd);
             }
+            // Update the leader lease when the leader commits the first empty entry
+            // during its term.
+            if self.is_leader() && term == self.raft_group.raft.term {
+                let now = clocktime::now_monotonic_raw_clocktime();
+                let next_expired_time =
```

Comment:
This will keep updating self.leader_lease_expired_time to `now` plus an election timeout. I think L530 has update the time properly, no need to do it again here.